### DISCUSSION TITLE
Basics // How to insert the document name, e.g. "invoice" the right way.

### DISCUSSION THREAD

**Discussion by LeHarry** [frappe/print_designer#168] [Q&A] [answered]
![grafik](https://github.com/frappe/print_designer/assets/160015230/ebcd6875-7768-4920-a44d-ca5cf2953410)
`This is the generic auto-translating document name.`

Q: How do you insert an auto-translating word like that, e.g. "Invoice" into Printdesigner?

**Comment by maharshivpatel** [comment, marked as answer]
use jinja and wrap it inside translate function like `{{_("Invoice")}}`
